docs/developers/testing.md

```diff
@@ -30,15 +30,20 @@ from unit tests to integration and functional tests.
 
 All of `napari` tests are located in folders named `_tests`. We keep our unit
 tests located in the individual folders with the modules or functions they are
-testing (e.g. the tests for the `Image` layer are located in a `_tests` folder
-alongside the `image.py` file). Our integration and functional tests are located in
+testing (e.g. the tests for the `Image` layer are located in
+`/napari/layers/image/_tests` alongside the `Image` layer code). Our integration and
+functional tests are located in
 the `napari/_tests` folder at the top of the repository.
 
 We also strive to unit test as much of our model file and utils code independently of
 our GUI code. These tests are located in the `napari/layers`, `napari/components`,
 and `napari/utils` folders. Our GUI code is tests in the `napari/_tests`,
-`napari/_qt`, `napari/_vispy` folders. The `napari/plugins` folder contains a mix
-of tests.
+`napari/_qt`, `napari/_vispy` folders. The tests in these three folders are ignored
+when we are testing in a headless environment (in continuous integration workflows).
```

Comment:
```suggestion
when we are testing in a headless environment without a Qt backend 
(for example in some of our continuous integration workflows).
```
I feel like the current copy makes it sound like all napari CI is headless and doesn't run these tests, but in fact not all of napari CI tests are headless.

Reply:
Good point, re-worded. Hopefully it's okay.